build/build.sh

```diff
@@ -71,5 +102,9 @@ docker rm -f "$squashfsContainerID"
 rm -f "$(pwd)/$tarFileName"
 
 squashfsAbsolutePath="$(pwd)/$squashfsFilename"
-# AzureDevOps specific way of passing an output variable to subsequent steps in the pipeline
-echo "##vso[task.setvariable variable=squashfsAbsolutePath;isOutput=true]$squashfsAbsolutePath"
+
+# If you want to promote the image directly to the caching server, run ./build.sh buildSquashfsAndPromote="true"
```

Comment:
Wouldn't 'promoteSquasfs' be clearer? This toggle only controls the promote part. The Squasfs will always be built.

Also, I'd extend this comment a bit like:

> this base squashfs will always be pushed into /dev/

Reply:
No, this actually skips the squashfs build, as we don't need it in most cases. This is what saves us time (and the reason to implement this story).